style.css

```diff
@@ -0,0 +1,665 @@
+  }
+
+  .logo {
+    margin-left: 150px;
+  }
+
+  .menu {
+    display: block;
+    float: right;
+    margin-right: 150px;
+  }
+
+  nav {
+    display: none;
+  }
+
+  .back_image {
+    background-image: url('Images/bigger_img.png');
+    background-position: fixed;
+    max-width: 100%;
+    margin-top: 56px;
+    margin-left: 30px;
+  }
+
+  .heading {
+    margin-left: 130px;
+    font-size: 48px;
+  }
+
+  .heading span {
+    display: contents;
+  }
+
+  .bio {
+    margin-left: 130px;
+    width: 70%;
+    font-size: 20px;
+  }
+
+  .links {
+    margin-left: 130px;
+    font-size: 16px;
+  }
+
+  .project_one {
+    flex-direction: row;
+  }
+
+  .project_one img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .project_two {
+    width: auto;
+    height: auto;
+    background: #fff;
+    border-radius: 16px;
+    border: 1px solid #def1e6;
+    padding: 16px;
+    display: flex;
+    justify-content: center;
+    flex-direction: row-reverse;
+  }
+
+  .project_two img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .name_des {
+    width: 450px;
+    margin: 10px 10px 10px 30px;
+  }
+
+  .nam {
+    font-size: 40px;
+  }
+
+  .list {
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .title_des {
+    font-size: 16px;
+    width: 100%;
+  }
+
+  .about {
+    flex-direction: row;
+  }
+
+  .about_one {
+    margin: 80px 30px 80px 80px;
+  }
+
+  .about_intro {
+    width: 399px;
+  }
+
+  .languages_skill {
+    margin-top: 80px;
+  }
+
+  .skill_bars {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .skill_bar {
+    width: 122px;
+    height: 100px;
+    margin: 10px;
+    padding: 10px;
+    flex-direction: column;
+  }
+
+  .skill_text {
+    font-size: 15px;
+    margin-left: 1px;
+  }
+
+  .work_skill h3 {
+    font-size: 15px;
+  }
+
+  .skill_bar img {
+    width: 48px;
+  }
+
+  .work_skill img {
+    width: 20px;
+  }
+
+  .resume1 {
+    background: transparent;
+    padding: 16px;
+    margin-top: 40px;
+    color: #6070ff;
+    border: 1px solid #6070ff;
+    border-radius: 16px;
+    font-weight: 500;
+    font-size: 25px;
+    cursor: pointer;
+    outline: none;
+    width: 10rem;
+  }
+
+  .resume1:hover {
+    background-color: #6b72b4;
+    color: white;
+  }
+
+  .resume1:active {
+    background-color: #2230d2;
+    color: white;
+    transform: translateY(4px);
+  }
+
+  .bottom_image {
+    background-image: url('Images/bottom_big.png');
+    background-size: cover;
+    max-width: 100%;
+  }
+
+  .contact_full {
+    text-align: center;
+  }
+
+  .contact_me {
+    font-family: 'Poppins', sans-serif;
+    font-size: 35px;
+    color: #fff;
+    text-align: center;
+    margin-top: 80px;
+    font-weight: 700;
+  }
+
+  .last_par {
+    font-weight: 400;
+    font-size: 20px;
+    width: 30%;
+    margin: 0 35%;
+  }
+
+  #input {
+    width: 50%;
+    border-radius: 0;
+  }
+
+  #textpanel {
+    width: 50%;
+    border-radius: 0;
+  }
+
+  .submit {
+    border-radius: 0;
+  }
```

Comment:
- [ ] Kindly make sure to add the `border-radius` on the `submit` button in order to meet the Figma design as much as possible.

  - ### YOUR DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/220857139-6f631a1b-c486-42fb-a1c5-e0460eed66da.png)

  - ### FIGMA DESIGN
  
   ![image](https://user-images.githubusercontent.com/87197412/220857208-7ae7ee42-447e-4b9e-b01f-56875f63d82e.png)